src/app/config/no-config/no-config.component.html

```diff
@@ -41,7 +41,7 @@
       <fa-icon [icon]="['fas', 'smile']"></fa-icon>
     </span>
   </div>
-  <div *ngIf="page === 1">
+  <div *ngIf="page === 3">
```

Comment:
I think these should be sorted again once everything is complete. Fine for now though, since the whole setup screen needs a revamp once all the changes are in.

Reply:
I need to go retest these I though I tested going all the way through and back 👍 

But that is why I didn't sort it I assume most of this will be gone in step 2.

Reply:
Leave that for now, We can go through this once the rest is done. There will be a couple of changes here I guess.